src/createButton.ts

```diff
@@ -12,11 +14,26 @@ export interface ButtonFactory extends ComposeFactory<Button, ButtonOptions> { }
 
 const createButton: ButtonFactory = createWidget
 	.mixin({
-		mixin: createFormFieldMixin
+		mixin: createFormFieldMixin,
+		aspectAdvice: {
+			before: {
+				getNodeAttributes(this: Button, overrides: VNodeProperties = {}) {
+					if (this.state.classes) {
+						const classes: { [index: string]: boolean; } = {};
+						this.state.classes.forEach((c) => classes[c] = true);
+						classes[ css.button ] = true;
+						overrides.classes = classes;
+					}
+
+					return [overrides];
+				}
+			}
+		}
 	})
 	.extend({
 		tagName: 'button',
-		type: 'button'
+		type: 'button',
+		classes: [ css.button ]
```

Comment:
I need to think about how this works... because I can see the desire to work here, but I need to check to see if this blows away if they are set in the state, like above.  I guess it must have worked for you.